src/components/settings/About.vue

```diff
@@ -21,6 +21,9 @@
                     <label><input v-model="settings.checkBeta" type="checkbox" :disabled="!settings.checkUpdate" @change="saveSettings()" /> {{ $t("Also check beta release") }}</label>
                 </div>
             </div>
+            <div class="mt-3">
```

Comment:
Let's add more space here, looks too connected to above otherwise.

```suggestion
            <div class="mt-5">
```